pkg/backingstore/backingstore.go

```diff
@@ -546,7 +547,11 @@ func RunCreateS3Compatible(cmd *cobra.Command, args []string) {
 		sigVer, _ := cmd.Flags().GetString("signature-version")
 		secretName, _ := cmd.Flags().GetString("secret-name")
 		mandatoryProperties := []string{"AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"}
-
+		u, _ := url.Parse(endpoint)
+		if u.Scheme == "http" {
+			sigVer = ""
```

Comment:
We should add here a printing that the signature-version v2 will be used.
We should also fail here if a `signature-version` is provided and it is v4

Reply:
`We should also fail here if a signature-version is provided and it is v4` -->> default value for `signature-version` is always V4, There is no way we can identify whether this value came from user or for default value.

Reply:
So, what is the meaning of sigVer = "" ?
If the meaning is that it will not be set, and as a result it will be V2, then when a user explicitly sets it to v4 and user HTTP we should fail.

Reply:
@liranmauda  So your suggestion is to make the property `signature-version` in CLI?

Reply:
My suggestion is to set the default to empty (""), in case we will get empty string we will always send v4 unless it's namespacestore s3 with http - than v2. only if we will get v4 by user request  - we will fail. @naveenpaul1 @liranmauda what do you think?